config/celery_app.py

```
@@ -15,3 +17,16 @@
 
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks()
+
+
+@task_retry.connect
+def sentry_log_retry(request=None, reason=None, einfo=None, **kwargs):
```

Ruff (0.14.11)

Unused function argument: `einfo`

(ARG001)